pkg/phases/configmapreloader/install.go

```diff
@@ -24,5 +25,12 @@ func Deploy(p *platform.Platform) error {
 	if err := p.CreateOrUpdateNamespace(constants.PlatformSystem, nil, nil); err != nil {
 		return err
 	}
-	return p.ApplySpecs("", "configmap-reloader.yaml")
+
+	if err := p.ApplySpecs("", "configmap-reloader.yaml"); err != nil {
+		return err
+	}
+
+	p.WaitForNamespace(constants.PlatformSystem, 60*time.Second)
```

Comment:
This might introduce some unintended delays if there are other resources in platform-system not healthy during a deployment that doesn't actually change configmap-reloader - The easiest fix would be to move the wait into the test vs the deployment